src/components/listOfQuestions/index.js

```diff
@@ -0,0 +1,71 @@
+import { connect } from 'react-redux';
+import { getQuestionsList } from '../../store/index.js';
+import { acceptQuestion, rejectQuestion } from '../../store/question.js';
+
+import './index.css';
+export const Question = ({
+	onAccept,
+	onReject,
+	question: { question, askee, status, id } = {},
+}) => {
+	return (
+		<div
+			className={'list-of-questions__question'}
+			data-testid={`question-${id}`}
+		>
+			<span className={'list-of-questions__question__question'}>
+				Question: {question}
+			</span>
+			<span className={'list-of-questions__question__askee'}>
+				Askee: {askee}
+			</span>
+			<span className={'list-of-questions__question__status'}>
+				Status: {status}
+			</span>
+			{status === 'Unanswered' ? (
+				<section>
+					Modify status:
+					<button onClick={() => onAccept(id)}>Accept</button>
+					<button onClick={() => onReject(id)}>Reject</button>
+				</section>
+			) : null}
+		</div>
+	);
+};
+
+let ListOfQuestions = ({
+	questions = [],
+	positionClass,
+	onAccept,
+	onReject,
+}) => {
+	return (
+		<div className={`${positionClass} list-of-questions`}>
+			{Array.isArray(questions) && questions.length > 0 ? (
```

Comment:
You shouldn't need the guards here because you have a default value, and your selector should always return an array (if it doesn't, fix the selector instead of adding a guard here). Ditto for the length check - that shouldn't be required, either.

Reply:
In general, your code should not need to explicitly check types. If it does, there may be a better way to do what you want.